lessons/basics/modules.md

```diff
@@ -194,9 +202,28 @@ import List, only: :functions
 import List, only: :macros
 ```
 
+It is also worth noting that import is lexical in scope.
+
+```elixir
+iex> defmodule MyModule do
+...>   def func_a do
+...>     import List, only: [duplicate: 2]
```

Comment:
Let's remove this.  I don't like it and I don't want to encourage this nor confuse folks.